src/Illuminate/Foundation/Console/ClearCompiledCommand.php

```diff
@@ -25,17 +25,22 @@ class ClearCompiledCommand extends Command {
 	 */
 	public function fire()
 	{
-		$compiledPath = $this->laravel->getCachedCompilePath();
-		$servicesPath = $this->laravel->getCachedServicesPath();
-
-		if (file_exists($compiledPath))
+		if (method_exists($this->laravel,'getCachedCompilePath'))
 		{
-			@unlink($compiledPath);
+			$compiledPath = $this->laravel->getCachedCompilePath();
```

Comment:
Do you mean that I should only have the following inside the method_exists if block:

$compiledPath = $this->laravel->getCachedCompilePath();
@unlink($compiledPath);

or something else?